docker-compose.yml

```diff
@@ -11,3 +12,32 @@ services:
       - "8546"
     user: root
     working_dir: /home/parity
+
+  plunger-list:
+    image: plunger
+    command: --rpc-host $ETH_RPC_URL --source parity_txqueue --list $ETH_FROM
+    secrets:
+      - keystore.json
+      - password.txt
+    environment:
+      - ETH_RPC_URL
+      - ETH_FROM
+
+  plunger-force:
+    image: plunger
+    command: |
+      --rpc-host $ETH_RPC_URL --source parity_txqueue --override-with-zero-txs \
+      --eth-key=key_file=/run/secrets/keystore.json,pass_file=/run/secrets/password.txt \
```

Comment:
would be nice to be able to use `plunger-force` with several accounts, not only one. If for `plunger-list` we can specify a different `$ETH_FROM` we should be able to specify which keystore / pwd to use (rather mount a directory with multiple keystores / passwords than a single one)

mind that if we just omit password within `--eth-key` e.g. `--eth-key=key_file=/run/secrets/keystore.json` then user will be asked to manually input password on command execution, which is safer from security pov

Reply:
`$ETH_FROM` is a string so that's easy, keystore might be a bit trickier because we have to get the file into the container, with either secret or volume. I'll have a think but also don't wanna spend too much time on this as it will probably be wrapped in a script in the sysadmin tools.

Reply:
Could let the user specify the local path to keystore:

```volumes:
    - ${KEYSTORE_PATH}:/path/to/container/app```

Are you thinking something like that?

Reply:
Yes, that should work